twilio.cabal

```diff
@@ -121,3 +121,12 @@ test-suite Tests
                        text,
                        transformers,
                        twilio
+
+test-suite integrationtests
+  hs-source-dirs:      test/unit
+  main-is:             Spec.hs
+  Type:                exitcode-stdio-1.0
+  other-modules:       DoesItWorkSpec
```

Comment:
What is "DoesItWorkSpec"?

Reply:
Sorry. This is a a vestige of me making sure everything was wired properly. I'll delete it.